examples/ola-recorder.cpp

```diff
@@ -68,6 +68,10 @@ DEFINE_uint32(stop, 0,
               "Time (milliseconds) in show file to stop playback at. If "
               "the show file is shorter, the last look will be held until the "
               "stop point.");
+DEFINE_s_string(start_trigger_channel, stc, "",
```

Comment:
```suggestion
DEFINE_string(start_trigger_channel, "",
```
The short option (stc as you had) needs to be a single character, so you can do `-u 1` for example. Given this option is somewhat more obscure, I'm not sure tying a letter to it makes sense, so the above just enables it as a long option only.